src/pykeen/nn/functional.py

```diff
@@ -1039,21 +1056,10 @@ def quat_e_interaction(
         The scores.
     """
     h, r, t = [x.view(*x.shape[:-1], -1, 4) for x in (h, r, t)]
```

Comment:
maybe make a function that wraps `x.view(*x.shape[:-1], -1, 4)` called `view_quaternion`